pubspec.lock

```diff
@@ -1584,15 +1600,24 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "3.1.2"
+  znn_ledger_dart:
+    dependency: "direct main"
+    description:
+      path: "."
+      ref: "feature/ledger"
+      resolved-ref: "3b11e8e3d46b403370e3888bc18a294781cbc0d9"
+      url: "https://github.com/kinggorrin/znn_ledger_dart.git"
+    source: git
+    version: "0.0.2"
   znn_sdk_dart:
     dependency: "direct main"
     description:
       path: "."
-      ref: d960c0bdc6dc553eaa75d5ddddb34cab530578b8
-      resolved-ref: d960c0bdc6dc553eaa75d5ddddb34cab530578b8
-      url: "https://github.com/zenon-network/znn_sdk_dart.git"
+      ref: "feature/ledger"
+      resolved-ref: d8e38c3a0adf05b8b25c6766645ed4c8b9b4de20
+      url: "https://github.com/kinggorrin/znn_sdk_dart.git"
```

Comment:
This should probably be pointing to the zenon-network repo. Not a blocker for this PR and can be fixed separately.